test_dbhandler.js

```diff
@@ -0,0 +1,8 @@
+const db = require('./routes/dbhandler');
```

Comment:
We can actually remove this file - this was just to illustrate how dbhandler.js was supposed to work.